### ISSUE TITLE
Bug: Failed to load config "plugin:react-hooks/recommended" to extend from.

### ISSUE THREAD

**Issue by WillSquire** [facebook/react#18520]
react-scripts version: 3.4.1

## Steps To Reproduce

1. Created project with CRA typescript
2. Added `.eslintrc.js`:
```
module.exports = {
  root: true,
  parser: '@typescript-eslint/parser',
  parserOptions: {
    tsconfigRootDir: __dirname,
    project: ['./tsconfig.json'],
    ecmaFeatures: {
      jsx: true,
    },
  },
  plugins: ['@typescript-eslint', 'jest', 'react', 'react-hooks', 'sonarjs'],
  extends: [
    'standard-with-typescript',
    'plugin:jest/recommended',
    'plugin:react/recommended',
    'plugin:react-hooks/recommended',
    'plugin:@typescript-eslint/recommended-requiring-type-checking',
    'plugin:sonarjs/recommended',
  ],
  settings: {
    react: {
      version: 'detect',
    },
  },
}
```
3. Added lints not included with `react-scripts` to `devDependencies`:
```
    "eslint-config-standard-with-typescript": "^15.0.1",
    "eslint-plugin-jest": "^23.8.2",
    "eslint-plugin-node": "11",
    "eslint-plugin-promise": "4",
    "eslint-plugin-sonarjs": "^0.5.0",
    "eslint-plugin-standard": "4"
```
4. Run with `npx eslint --fix --ext .js,.jsx,.ts,.tsx .`

Worked around adding rules individually:
```
rules: {
    'react-hooks/rules-of-hooks': 'error',
    'react-hooks/exhaustive-deps': 'warn',
},
```

**Comment by dougbacelar** [comment]
Perhaps a duplicate of: https://github.com/facebook/react/issues/18208

Check out the first comment from @threepointone 
> The commit that landed that behaviour (#14762) hasn't been released yet. We could probably do a release sometime, but the workaround right now is to copy paste the recommended config. I'll leave this open till we do.

**Comment by WillSquire** [comment]
Yep, sounds the same to me! Thanks @dougbacelar
